### PR TITLE
Added an endpoint to provide the user with a downloadable certificate of analysis coa link.

### PR DESCRIPTION
**Pull request by vigilantee**
Task: https://bloomstack.com/desk#Form/Task/TASK-2020-01283

### CODE REVIEW

File: erpnext/stock/doctype/batch/batch.py [hunk 1]
```diff
@@ -330,3 +330,9 @@ def get_active_batch(item_code):
 	active_batch = frappe.get_all("Batch", {"item": item_code, "display_on_website": 1})
 	active_batch = active_batch[0] if active_batch else {}
 	return active_batch
+
+@frappe.whitelist()
+def get_active_batch_coa(item_code):
+	active_batch = frappe.get_all("Batch", filters={"item": item_code, "display_on_website": 1}, fields=["name", "certificate_of_analysis"])
```
Comment (Alchez) [inline erpnext/stock/doctype/batch/batch.py]: Add the `fields` key to the `get_active_batch` function above instead, let's not make multiple endpoints for this. If you want, rename the function to `get_active_batch_details`

**Review by Alchez — CHANGES_REQUESTED** [review]
You have whitespace changes in your JSON file, please remove them

File: erpnext/stock/doctype/batch/batch.py [hunk 2]
```diff
@@ -326,7 +326,11 @@ def save_thc_cbd(batch_no, thc, cbd):
...
 
 @frappe.whitelist()
-def get_active_batch(item_code):
-	active_batch = frappe.get_all("Batch", {"item": item_code, "display_on_website": 1})
+def get_active_batch(item_code, coa=False):
+	if(coa):
+		fields = ["name", "certificate_of_analysis"]
+	else:
+		fields = ["name"]
+	active_batch = frappe.get_all("Batch", filters={"item": item_code, "display_on_website": 1},  fields=fields )
```
Comment (Alchez) [inline erpnext/stock/doctype/batch/batch.py]: You might want to get more fields then

```suggestion
def get_active_batch(item_code):
	fields = ["name", "item", "item_name", "stock_uom", "thc", "cbd", "certificate_of_analysis"]
	active_batch = frappe.get_all("Batch", filters={"item": item_code, "display_on_website": 1},  fields=fields)
```

Comment (vigilantee) [reply erpnext/stock/doctype/batch/batch.py]: Done.

**Review by Alchez — DISMISSED** [review]

**Comment by Alchez** [comment]
Moved to #470

**Pull request closed without merging**